components/Tabs/Tabs.js

```diff
@@ -2,37 +2,41 @@
 class TabItem {
   constructor(element) {
     // attach dom element to object. Example in Tabs class
+    this.element = element;
   }
 
   select() {
-    // should use classList
+    this.element.classList.add('Tabs__item-selected')
```

Comment:
Try to keep your formatting consistent (e.g., I notice lines 9 and 13 do not end with semicolon, while the rest do. )